cloud/amazon/ec2_ami.py

```diff
@@ -44,7 +44,7 @@
     default: 300
   state:
     description:
-      - create or deregister/delete image
+      - create (present), register (register), deregister/delete image (absent)
```

Comment:
Yeah, maybe those could be aliases. Absent/deregistering an image is the same operation for all AMIs as far as I know. "Creating" (turning an instance into AMI) and "registering" (importing a new AMI from S3) are two different things though.